### PR TITLE
Suppress Ruby 2.4 Fixnum warnings

### PR DESCRIPTION
**Pull request by barthez**
I'm trying to update Unparser gem to support Ruby 2.4, but there is a complicated chain of dependencies. 

I will try to update devtools and mutant, so I can make Unparser depend on Parser 2.4 and support new language features (I believe there is only one: multiple assignments in if statement).

We need to update SimpleCov - because 0.12.0 was using Fixnum as well, reek 4.5 depends on parser v2.3.0.

### CODE REVIEW

File: lib/devtools/config.rb [hunk 1]
```diff
@@ -109,21 +109,16 @@ class Reek < self
...
-    #
-    # rubocop:disable Lint/UnifiedInteger
     class Flay < self
       FILE             = 'flay.yml'.freeze
       DEFAULT_LIB_DIRS = %w[lib].freeze
       DEFAULT_EXCLUDES = %w[].freeze
 
-      attribute :total_score, [Fixnum]
-      attribute :threshold,   [Fixnum]
+      attribute :total_score, [0.class]
```
Comment (mbj) [inline lib/devtools/config.rb]: I think this is a good solution for 2.3 vs 2.4, once we drop < 2.4 we'll have to go to `Integer`.

**Comment by mbj** [comment]
@barthez Thx. Code wise I'm fine with this. Will merge on green. @backus When you see this green earlier than me and agree feel free to merge also.

**Comment by mbj** [comment]
Mhh, circle CI integration appears to be dead.

**Comment by barthez** [comment]
@mbj It seems we are having troubles with Flay:
```
Flay total is now 115, but expected 112
```
I'm not familiar with Flay at all, can I just increase `total_score` to 115?

**Comment by mbj** [comment]
> I'm not familiar with Flay at all, can I just increase total_score to 115?

Yes. This change is within the acceptable standard derivation.

**Comment by mbj** [comment]
@barthez Do you mind to squash all these together? This way we have one atomic good change, rater than 3 changes that first put the repo in a bad state that with the last commit is resolved.

**Comment by barthez** [comment]
@mbj Done 👍

**Comment by mbj** [comment]
@barthez Released as `0.1.18`.

**Pull request merged**